pdcupdater/handlers/modules.py

```diff
@@ -116,16 +120,19 @@ def create_unreleased_variant(self, pdc, body):
         runtime_deps = []
         for dep, ver in mmd.requires.items():
             if ver is not None:
-                runtime_deps.append("{} >= {}".format(dep, ver))
+
+                runtime_deps.append({"dependency": "%s-%s" % (dep, ver)})
             else:
-                runtime_deps.append(dep)
+                runtime_deps.append({"dependency": dep})
+        log.debug(runtime_deps)
 
         build_deps = []
         for dep, ver in mmd.buildrequires.items():
             if ver is not None:
-                build_deps.append("{} >= {}".format(dep, ver))
+                build_deps.append({"dependency": "%s-%s" % (dep, ver)})
             else:
-                build_deps.append(dep)
+                build_deps.append({"dependency": dep})
```

Comment:
Thanks @nphilipp.  I'm breaking the PR apart into multiple other PRs.

I will not copy this part over.